deploy/clowdapp.yaml

```diff
@@ -549,6 +549,29 @@ objects:
         WHERE rule_results.result IN ('fail', 'error', 'unknown', 'fixed')
         GROUP BY "rules"."ref_id", "accounts"."org_id";
 
+- apiVersion: metrics.console.redhat.com/v1alpha1
+  kind: FloorPlan
+  metadata:
+    name: compliance-hms
+  spec:
+    database:
+      secretName: ${FLOORIST_DB_SECRET_NAME}
+    objectStore:
+      secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+    logLevel: ${FLOORIST_LOGLEVEL}
+    suspend: ${{FLOORIST_SUSPEND}}
+    queries:
+    - prefix: hms_analytics/compliance/tests
+      query: >-
+        SELECT
+          "v2_test_results"."start_time" AS "start_time",
```

Comment:
@romanblanco can you please remind me what is the `start_time`.

@shaded-enmity are there more timestamps that the query could include?

Reply:
`start_time` is the start time of scanning

Reply:
I'd include also `end_time` as well. Just in case.